dockerfiles/Dockerfile.oracle

```diff
@@ -0,0 +1,74 @@
+FROM ghcr.io/oracle/oraclelinux:8-slim as builder-base
+
+RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log
+
+ENV REPOURL=https://yum.oracle.com/repo/OracleLinux/OL8/appstream/x86_64/getPackage
+RUN microdnf update -y \
+    && microdnf --enablerepo=ol8_codeready_builder install -y \
+    curl \
+    ca-certificates \
+    cmake \
+    git \
+    make \
+    gcc \
+    gcc-c++ \
+    tar \
+    openssl openssl-libs openssl-devel \
+    cyrus-sasl-lib cyrus-sasl-devel \
+    pkg-config \
+    systemd-devel \
+    zlib-devel \
+    libpq-13.5-1.el8.x86_64 \
+    libicu libicu-devel \
+    postgresql-devel \
+    flex \
+    bison \
+    libyaml-devel \
+    && rpm -ivh ${REPOURL}/postgresql-13.5-1.module+el8.5.0+20464+55827c52.x86_64.rpm \
```

Comment:
FluentBit installs it https://github.com/verrazzano/fluent-bit/blob/a07d45ab29c697c02722a3ee7b02da2206389d58/dockerfiles/Dockerfile#L52, I don't know the exact usage, but I couldn’t get the image to work without it. Probably it's for https://github.com/verrazzano/fluent-bit/tree/master/plugins/out_pgsql to make the plugin work.

To do the oracle bfs, I used centos dockerfile as a reference https://github.com/verrazzano/fluent-bit/blob/master/dockerfiles/Dockerfile.centos7

Reply:
I think postgresql should be installed using [DNF modules and application stream](https://docs.oracle.com/en/operating-systems/oracle-linux/software-management/sfw-mgmt-InstallSoftwareonOracleLinux.html#appstream) and not rpm.

Reply:
I tried microdnf and yum to install these, but the naming of these packages is weird and the package managers aren't able to locate these. Output of `yum install`
```
Error: Unable to find a match: postgresql-13.5-1.module+el8.5.0+20464+55827c52.x86_64 postgresql-server-13.5-1.module+el8.5.0+20464+55827c52.x86_64 postgresql-server-devel-13.5-1.module+el8.5.0+20464+55827c52.x86_64
```
So that's why I'm relying on rpm install. I need to install a specific version of these packages, if there's a way to install these using package managers then I'll use it.

Reply:
Try `dnf module list postgresql`

Reply:
Updated, now installing `postgresql` using `microdnf `